air/src/lib.rs

```diff
@@ -267,11 +271,15 @@ impl Serializable for PublicInputs {
         target.write(self.stack_inputs.values());
 
         // write program outputs.
```

Comment:
nit: Should probably be changed to stack outputs

Reply:
Fixed!